structures/Channel.ts

```diff
@@ -0,0 +1,56 @@
+import Base from './Base';
+//import User from './User'         doesn't exist yet
+
+// https://discord.com/developers/docs/resources/channel
```

Comment:
Please don't include any unclean comments here

Reply:
I just removed that